cmd/outline-ss-server/metrics.go

```diff
@@ -28,35 +28,136 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const namespace = "shadowsocks"
-
 // `now` is stubbable for testing.
 var now = time.Now
 
-type outlineMetrics struct {
-	ipinfo.IPInfoMap
-	*tunnelTimeCollector
+type tcpCollector struct {
+	probes               *prometheus.HistogramVec
```

Comment:
Perhaps add a comment that new metrics need to be added to `newTCPCollector`, `Describe` and `Collect`.